### PR TITLE
Minor code cleanup

### PR DESCRIPTION
**Pull request by kelvinqian00**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -15,7 +15,7 @@ jobs:
         uses: actions/checkout@v4
 
       - name: Get an env
-        uses: yetanalytics/action-setup-env@v2.0.0-SNAPSHOT
+        uses: yetanalytics/action-setup-env@v1
```
Comment (milt) [inline .github/workflows/ci.yml:18]: isn't it v1.1.0 now?

Comment (kelvinqian00) [reply .github/workflows/ci.yml:18]: Yes, but I changed the contents of v1, as is the practice with minor version updates for GH actions.

Comment (kelvinqian00) [reply .github/workflows/ci.yml:18]: To be clear the v1.0.0 and v1.1.0 tags remain immutable.

Comment (milt) [reply .github/workflows/ci.yml:18]: ah ok

**Review by milt — APPROVED** [review]

**Pull request merged**